frontend/src/nav/Menu.js

```diff
@@ -1,15 +1,28 @@
 import styled from "styled-components/macro";
-import logo from "../pictures/logo.png";
+import logo from "../images/logo.png";
+import { NavLink } from "react-router-dom";
 
-export default function Menu({ open }) {
+export default function Menu({ open, setOpen }) {
   return (
     <StyledMenu open={open}>
-      <a href="/">Sympathise</a>
-      <a href="/">Notfallhilfe</a>
-      <a href="/search">Beratungsstelle finden</a>
-      <a href="/">Gemerkt</a>
-      <a href="/">Login</a>
-      <a href="/">Links</a>
+      <NavLink to="/" onClick={() => setOpen(!open)}>
```

Comment:
Following the above suggestion, you could then write here:

```suggestion
      <NavLink to="/" onClick={setOpen}>
```
```suggestion
      <NavLink to="/" onClick={setOpen}>
```

and so on..